openstack/cc3test/templates/cronjob-certificate.yaml

```
@@ -4,7 +4,7 @@ metadata:
   name: cc3test-certificate
   namespace: {{ .Values.cc3test.namespace }}
 spec:
-  schedule: "*/30 8-15 * * *"
+  schedule: "0 1 * * *"
```

Do something odd here, not exactly on top of the hour.

```suggestion
  schedule: "27 1 * * *"
```

---

@viennaa updated cronjob schedule to run twice in a day (1:27AM and 13:27 PM) in new commit request